utils/variables_test.go

```diff
@@ -32,9 +33,6 @@ func TestJSONTags(t *testing.T) {
 }
 
 func TestValidNameRegex(t *testing.T) {
-	// validNameRegex is the pattern that matches to a valid name.
-	// The name must begin with a letter i.e. [a-zA-Z],
-	// and the body can only contain 0 or more numbers, letters, dashes and underscores i.e. [-\w]*.
```

Comment:
duplicate comment that provided no additional info that wasn't already above the `ValidNameRegex`